src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -272,6 +272,7 @@ def _executeCondorCommand(self, cmd, keepTokenFile=False):
             htcEnv = {
                 "_CONDOR_SEC_CLIENT_AUTHENTICATION_METHODS": "SCITOKENS",
                 "_CONDOR_SCITOKENS_FILE": self.tokenFile.name,
+                "_CONDOR_AUTH_SSL_CLIENT_CADIR": os.environ.get("X509_CERT_DIR", "/etc/grid-security/certificates"),
```

Comment:
I guess this should use `getCAsLocation()` rather than relying on the variable.

Reply:
Indeed